pkgs/native_toolchain_c/test/cbuilder/cbuilder_cross_android_test.dart

```diff
@@ -17,20 +17,24 @@ void main() {
     Target.androidArm64,
     Target.androidIA32,
     Target.androidX64,
+    // TODO(rmacnak): Enable when stable NDK 27 is available.
+    // Target.androidRiscv64,
```

Comment:
@rmacnak-google Can this PR be landed or are we blocked on having NDK 27?

It looks like that will be only in the new year: https://github.com/android/ndk/wiki#ndk-r27-lts

Reply:
We can land this if you're okay with leaving this test disabled, otherwise we need to wait for NDK 27.

Reply:
Okay lets land it then 👍  The changes eyeballed look good.